### ISSUE TITLE
Article: How to create and use a screen reader text class

### ISSUE THREAD

**Issue by RachelRVasquez** [a11yproject/a11yproject.com#375]
Hi there!

I'd like to contribute by writing an article on implementing and using a screen reader text class.

Reference: [Invisible Content Web AIM](http://webaim.org/techniques/css/invisiblecontent/).

I'd like to provide some examples like an icon font and links that open in a new tab/window where a screen reader text class can really increase accessibility.

You can see some examples of my a11y writing [here on my blog](http://rachievee.com/category/wordpress/accessibility-series/).

Thanks! Looking forward to writing. :-)

**Comment by RachelRVasquez** [comment]
I've created a gist plain text version of the article here:

https://gist.github.com/RachelRVasquez/2717f04df94ded23ab0b

It doesn't have any formatting. I'm thinking of splitting this into two articles instead - one for creating the class, another for actually using the class? Open for suggestions. :-)

**Comment by davatron5000** [comment]
Hi @RachelRVasquez thanks for helping and wanting to contribute. I read through your article and it seems very similar in technique to an article already existing on the site:

http://a11yproject.com/posts/how-to-hide-content/

I'm not sure we can have two articles demonstrating the same technique. Bummer. I do very much like your friendly writing style thought. The way you build up and educate on the "why" is a very good way for people to learn. We probably tend to be more cut-and-dry on the project, for better or for worse.

I'm not sure where you'd like to go from here. I'd love to have your help on the project. Just a thought: There are ~11 issues tagged "Article" on the site looking for feedback, vetting, or approval.

https://github.com/a11yproject/a11yproject.com/labels/Article

Any interest jumping in there and helping some of those get out the door?  And again, thanks so much for wanting to help. I personally really appreciate it.

**Comment by joe-watkins** [comment]
Welcome aboard @RachelRVasquez :)

**Comment by RachelRVasquez** [comment]
@davatron5000 Ah you're right! I looked at the article some time late at night and totally didn't see the css there - darn! Well perhaps the examples can still be of use? Maybe we can use them as patterns or quick tips?

I'll take a look at the list of articles people have there and see what else there is I can do then. :-) I can still use this gist for my blog so yay!

Thanks @joe-watkins happy to be here and contribute.

**Comment by RachelRVasquez** [comment]
@davatron5000 I've had a look and the Accessible Links looks like a good one to grab! 

https://github.com/a11yproject/a11yproject.com/issues/235

Are we waiting on more expansion for the "Placeholder is not a label" idea? 

https://github.com/a11yproject/a11yproject.com/issues/170

**Comment by joe-watkins** [comment]
@RachelRVasquez if you need some code examples or anything I've [rambled about the topic of Placeholders](https://joe-watkins.io/accessibility/please-stop-using-placeholders-as-labels/) on my blog.. feel free to steal anything if it helps.

**Comment by RachelRVasquez** [comment]
@joe-watkins Oh cool, thanks for the link! :-)

**Comment by svinkle** [comment]
This issue has become "[stale](https://github.com/a11yproject/a11yproject.com/blob/gh-pages/CONTRIBUTING.md#stale-issues)" — @RachelRVasquez, feel free to open a [new issue](https://github.com/a11yproject/a11yproject.com/issues/new) if you'd like to continue.
